man/docker-images.1.md

```diff
@@ -103,6 +103,24 @@ which displayed different visualizations of the image data. Docker core removed
 this functionality in the 1.7 version. If you liked this functionality, you can
 still find it in the third-party dockviz tool: https://github.com/justone/dockviz.
 
+## Listing images in a desired format
+
+You can use the `--format` parameter to list only the image attributes required
+and in the order required. For example, if the image ID, Tag, and REPOSITORY 
+are required, in that order, then the following command is used:
+
+    docker images --format "{{.ID}}: {{.Tag}}\t{{.Repository}}"
+
+It produces output similar to :
+
+    11a5207645d4: latest	docker.io/fedora
+    4211be3df429: latest	docker.io/ubuntu
+    f7c419c1256d: latest	ipbabble/fedora-httpd
+    ce859f6e9f3d: latest	fedora-sshd
+    f9d5dee79539: latest	docker.io/kubernetes
+
+Valid template placeholders are listed above.
```

Comment:
Actually; I noticed we already have a section describing this functionality in the CLI reference. I think it's better to just copy that part in the man page; https://github.com/docker/docker/blob/master/docs/reference/commandline/images.md#formatting

Note that it needs slight modification, because the `{% raw %}` and `{% endraw %}` markers are not needed for the man pages